crates/topos-tce-broadcast/src/double_echo/mod.rs

```diff
@@ -30,30 +30,39 @@ pub struct DeliveryState {
 }
 
 pub struct DoubleEcho {
-    last_pending_certificate: PendingCertificateId,
-    pub(crate) params: ReliableBroadcastParams,
+    // external communication
```

Comment:
I think we can use doc comments here?

Reply:
It is more about grouping argument together that documenting a particular one :)

Reply:
On many places we document `pub struct` and each field. It also generates a nice documentation at some point as a bonus. I think we should get into the habit of writing a short sentence (or more) what the struct and the fields are for?